opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -219,6 +219,19 @@ namespace Opm
             std::cout << "*          For more information, see https://opm-project.org         *\n";
             std::cout << "*                                                                    *\n";
             std::cout << "**********************************************************************\n\n";
+
+            int threads = 1;
+            int mpiSize = 1;
+
+#ifdef _OPENMP
+            const int threads = omp_get_max_threads();
```

Comment:
This will shadow the existing variable, why not just
```
threads = omp_get_max_threads();
```

Reply:
amended and force pushed.